app/views/collections/_form.html.erb

```diff
@@ -110,6 +110,19 @@
               presence: false, maximum_length: ArchiveConfig.SUMMARY_MAX) %>
         <%= generate_countdown_html("collection_description", ArchiveConfig.SUMMARY_MAX) %>
       </dd>
+
+      <dt><%= collection_form.label :tag_string, ts("Collection Tags") %></dt>
+      <dd>
+        <%= collection_form.text_field :tag_string, autocomplete_options('tag') %>
+        <p class="footnote">
+          <%= ts("Enter up to %{max} tags to describe the content of your collection.", max: ArchiveConfig.COLLECTION_TAGS_MAX) %>
+        </p>
+      </dd>
+
+      <dt><%= collection_form.check_box :multifandom %></dt>
+      <dd>
+        <%= collection_form.label :multifandom, ts("Use this if your collection is not fandom-specific.") %>
+      </dd>
```

Comment:
It looks like the label text and footnote got mixed up here. The label should be "Multifandom" and there should be a footnote saying "Use this if your collection is not fandom-specific." 

This footnote and the one above also need to be associated with the fields they describe using ARIA, like so:
https://github.com/otwcode/otwarchive/blob/5f9478beb2d9648d5fae6e85662c389c6bc8c989/app/views/abuse_reports/new.html.erb#L105-L110